client/state/sharing/publicize/publicize-actions/reducer.js

```diff
@@ -97,11 +106,23 @@ export const editingSharePostAction = createReducer( {}, {
 		( state, { siteId, postId, actionId } ) => updateDataForPost( true, state, siteId, postId, actionId ),
 } );
 
+export const schedulingSharePostActionStatus = createReducer( {}, {
```

Comment:
These reducers could use some tests to show what they do. `updateDataForPost` is a little confusing because it does not say in the name what it does (what is the "Data" that's updated, and how is it updated?). Often when a function is hard to name, it might be a good idea to split it into smaller sub-functions.